tests/integration/test_authd/force_options/data/test_cases/after_registration_time.yaml

```diff
@@ -0,0 +1,70 @@
+-
```

Comment:
It should be useful to add an id field without spaces and symbols. This will help us to Individually identify every tcase